<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>128</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Centro de custo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>341</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Novo Centro de Custo</string>
    </property>
    <widget class="QLineEdit" name="txtCentro">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>301</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSalvar">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>72</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Salvar</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icones/floppy-disk_1311670.png</normaloff>icones/floppy-disk_1311670.png</iconset>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Centro de custo</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
